.mailmap

```diff
@@ -128,35 +157,48 @@ Kelsey Breseman <ifoundthemeaningoflife@gmail.com>
 Koichi Kobayashi <koichik@improvement.jp>
 Kris Kowal <kris.kowal@cixar.com>
 Kyle Robinson Young <kyle@dontkry.com>
+Lakshmi Swetha Gopireddy <lakshmiswethagopireddy@gmail.com> LAKSHMI SWETHA GOPIREDDY <lgopired@in.ibm.com>
 Luke Bayes <lbayes@patternpark.com>
 Maciej Małecki <maciej.malecki@notimplemented.org> <me@mmalecki.com>
 Malte-Thorben Bruns <skenqbx@gmail.com> <skenqbx@googlemail.com>
+Malte-Thorben Bruns <skenqbx@gmail.com> skenqbx <skenqbx@gmail.com>
 Marcin Cieślak <saper@marcincieslak.com>
 Marcin Cieślak <saper@marcincieslak.com> <saper@saper.info>
 Marcin Zielinski <marzelin@gmail.com> marzelin <marzelin@gmail.com>
 Marti Martz <thalamew@q.com>
 Martial James Jefferson <martial.jefferson@gmail.com>
+Matthias Bastian <dev@matthias-bastian.de> piepmatz <piepmatz@users.noreply.github.com>
 Mathias Buus <mathiasbuus@gmail.com> <m@ge.tt>
 Mathias Pettersson <mape@mape.me>
 Matthew Lye <muddletoes@hotmail.com>
 Michael Bernstein <michaelrbernstein@gmail.com>
+Michael Dawson <michael_dawson@ca.ibm.com> <mdawson@devrus.com>
 Michael Wilber <gcr@sneakygcr.net>
-Michaël Zasso <targos@protonmail.com>
+Michaël Zasso <targos@protonmail.com> <mic.besace@gmail.com>
+Michael-Rainabba Richardson <rainabba@gmail.com> rainabba <rainabba@gmail.com>
 Micheil Smith <micheil@brandedcode.com> <micheil@yettobebranded.net>
 Micleusanu Nicu <micnic90@gmail.com>
 Mikael Bourges-Sevenier <mikeseven@gmail.com> <msevenier@motorola.com>
+Minqi Pan <pmq2001@gmail.com> P.S.V.R <pmq2001@gmail.com>
+Minwoo Jung <jmwsoft@gmail.com> JungMinu <jmwsoft@gmail.com>
 Miroslav Bajtoš <miroslav@strongloop.com> <miro.bajtos@gmail.com>
 Mitar Milutinovic <mitar.git@tnode.com>
 Myles Borins <myles.borins@gmail.com> <mborins@us.ibm.com>
 Myles Borins <myles.borins@gmail.com> <mylesborins@google.com>
 Nebu Pookins<nebu@nebupookins.net>
+Netto Farah <nettofarah@gmail.com> nettofarah <nettofarah@gmail.com>
 Nicholas Kinsey <pyrotechnick@feistystudios.com>
 Nikolai Vavilov <vvnicholas@gmail.com>
+Noah Rose Ledesma <noahroseledesma@seattleacademy.org> Noah Rose <noahroseledesma@seattleacademy.org>
+Noah Rose Ledesma <noahroseledesma@seattleacademy.org> <NoahRoseLedesma@seattleacademy.org>
 Onne Gorter <onne@onnlucky.com>
 Paul Querna <pquerna@apache.org> <paul@querna.org>
+Pedro Lima <pvsousalima@gmail.com> Pedro Victor <pvsousalima@gmail.com>
+Pedro Lima <pvsousalima@gmail.com> Pedro Lima <pvsousalima@gmail.com>
```

Comment:
Is this line needed?

Reply:
It is, but I made a typo. It's a version of the name with lowercase "L". Weird that it was working with that mistake...